Lib/test/test_capi/test_opt.py

```diff
@@ -792,48 +792,44 @@ def testfunc(n):
 
     def test_float_add_constant_propagation(self):
         def testfunc(n):
-            a = 1.0
```

Comment:
This is a limitation of loop-based traces. We should start projecting traces from function entries as well, but I'll leave that for Mark or someone else to do in the future.